.github/workflows/lint.yml

```diff
@@ -7,22 +7,21 @@ on:
 
 jobs:
   lint:
-
     runs-on: ubuntu-22.04
 
     steps:
-    - uses: actions/checkout@v4
-    - name: Set up Python 3.x
-      uses: actions/setup-python@v5
-      with:
-        python-version: '3.x'
-    - name: Update pip
-      run: pip install --upgrade pip
-    - name: Install black and pylint
-      run: pip install black~=24.4 pylint~=3.2
-    - name: Check files are formatted with black
-      run: |
-        black --check .
-    - name: Run pylint
-      run: |
-        pylint */
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+      - name: Update pip
+        run: pip install --upgrade pip
+      - name: Install black and pylint
+        run: pip install black pylint
+      - name: Check files are formatted with black
+        run: |
+          black --check .
```

Comment:
It would be great if you could add a ruff check here, see for example https://github.com/CQCL/pytket-quantinuum/pull/497

Reply:
Done